pkg/email/provider.go

```diff
@@ -0,0 +1,94 @@
+package email
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"strings"
+
+	"github.com/go-phorce/dolly/fileutil"
+	"github.com/go-phorce/dolly/xlog"
+	"github.com/juju/errors"
+	"gopkg.in/yaml.v2"
+)
+
+var logger = xlog.NewPackageLogger("github.com/ekspand/trusty/pkg", "email")
+
+// Client exposes send email operations
+type Client interface {
+	SendUserEmail(to, code string) error
```

Comment:
I think it should have only generic method, something like
`Send(from , to, message)`